### PR TITLE
Feature/vps service update

### PR DESCRIPTION
**Pull request by whispyy**
- Improve vps services imported to be faster and be more readable.
- Uses of ServiceHelper.
- Dashboard Controller now use ControllerHelperRequest
- BackupStorage Controller now use ControllerHelperRequest

### CODE REVIEW

File: client/app/vps/backup-storage/vps-backup-storage.controller.js [hunk 1]
```diff
@@ -1,70 +1,38 @@
...
-                            value: 0
-                        };
-                    }
-                }
-                if (!backupInfo.activated) {
-                    this.loadVps();
+        this.backup.load();
+        this.info.load()
+            .then(() => {
+                if (!this.info.data.activated) {
```
Comment (JDownloader) [inline client/app/vps/backup-storage/vps-backup-storage.controller.js]: It's a bit hard to know what's inside this.info.data.activated, we don't have a clue why the vps is only loaded if the backup option is deactivated

File: client/app/vps/backup-storage/vps-backup-storage.controller.js [hunk 2]
```diff
@@ -1,70 +1,38 @@
...
-        };
-        this.backup = {};
-        this.vps = {};
-
     }
 
-    $onInit () {
-        this.loadInformation();
-        this.loadBackup();
+    initLoaders () {
```
Comment (JDownloader) [inline client/app/vps/backup-storage/vps-backup-storage.controller.js]: I personally found ControllerHelper bloated and a result of our misunderstanding of normal angular services. This change is change is a bit the same of what was previously there and introduce a new way to initialise, which is missing the point of using a framework.

File: client/app/vps/import/vps.service.js [hunk 3]
```diff
@@ -1053,28 +1023,33 @@ angular.module("managerApp").service("VpsService", [
...
+                        }
+                    }
+                    return backupInfo;
+                })
+                .catch(ServiceHelper.errorHandler());
         };
 
         this.getBackupStorageTab = function (serviceName, count, offset) {
             vpsTabBackupStorageCache.removeAll();
-            return this.getSelectedVps(serviceName).then(function (vps) {
```
Comment (JDownloader) [inline client/app/vps/import/vps.service.js]: +1

File: client/app/vps/import/vps.service.js [hunk 4]
```diff
@@ -1053,28 +1023,33 @@ angular.module("managerApp").service("VpsService", [
         // BackupStorage ------------------------------------------------------------------------------------
 
         this.getBackupStorageInformation = function (serviceName) {
-            return this.getSelectedVps(serviceName).then(function (vps) {
```
Comment (JDownloader) [inline client/app/vps/import/vps.service.js]: +1

File: client/app/vps/import/vps.service.js [hunk 5]
```diff
@@ -907,50 +887,40 @@ angular.module("managerApp").service("VpsService", [
...
+                .catch(() => {
+                    return { state: "disabled" };
                 });
         };
 
         this.getTabVeeam = function (serviceName, state, forceRefresh) {
             if (forceRefresh) {
                 resetTabVeeam();
             }
-            return self.getSelectedVps(serviceName).then(function (vps) {
```
Comment (JDownloader) [inline client/app/vps/import/vps.service.js]: +1

File: client/app/vps/backup-storage/vps-backup-storage.controller.js (hunk 1 above)
Comment (whispyy) [reply client/app/vps/backup-storage/vps-backup-storage.controller.js]: vps is loaded only in that case to know if we can order it or not. Any idea ?

File: client/app/vps/backup-storage/vps-backup-storage.controller.js (hunk 2 above)
Comment (whispyy) [reply client/app/vps/backup-storage/vps-backup-storage.controller.js]: Tried to apply what have been made to make our life easier. Controller Helper allows us to win in lisibility in my opinion and it handles loaders.

**Pull request merged**